api.py

```diff
@@ -282,7 +282,7 @@ def cs_paged_query(q: str, resume: Optional[str], expanded: Optional[bool], sort
     query.update({
         "size": _validate_page_size(page_size or config["maxpage"]),
         "track_total_hits": False,
-        "sort": [{final_sort_field: final_sort_order}]
+        "sort": {final_sort_field: {"order": final_sort_order, "format": "basic_date_time_no_millis"}},
```

Comment:
Concern:
1. query pages divided by sort on a single column with a sort that is "unstable" (returns a different ordering each time) seems like it would be non-deterministic (could miss or duplicate items).  I have no idea whether ES "stabilizes" sorted results in any way.
2. using a secondary sort (on `_id`?) could provide "stability"??